api/oc_ri.c

```diff
@@ -1689,22 +1689,35 @@ oc_ri_free_client_cbs_by_mid(uint16_t mid)
 }
 
 void
-oc_ri_free_client_cbs_by_endpoint(const oc_endpoint_t *endpoint)
+oc_ri_free_client_cbs_by_mid(uint16_t mid)
+{
+  oc_ri_free_client_cbs_by_mid_v1(mid, OC_REQUEST_TIMEOUT);
```

Comment:
Wouldn't `OC_CANCELLED` be a better default than a timeout? (I've already changed the value from `OC_STATUS_SERVICE_UNAVAILABLE` to `OC_REQUEST_TIMEOUT`, so I don't think changing the code value is a problem).

Reply:
Ok. I didn't know it. Fixed